common/package.json

```diff
@@ -7,7 +7,8 @@
     "lint": "eslint ./src --ext .js,.ts,.jsx,.tsx",
     "lint:fix": "eslint ./src --fix --ext .js,.ts,.jsx,.tsx",
     "prettier": "prettier --write ./src",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
I'm having trouble running this, too.  Am I doing it wrong?
![image](https://user-images.githubusercontent.com/8978655/150022282-8c947dde-7420-44da-82c1-7f0b65080ba2.png)

Reply:
Hm. It works for me. Maybe rerun refresh_common? If you still have issues I'll look at it with you tomorrow

Reply:
OK, I dug in a little deeper.  The issue is that the common/tests folder does not exist in the repo.  If that folder doesn't exist then I get the error in the screenshot above and the tests won't run.  I think that git will normally not commit an empty folder so perhaps you could [do a workaround](https://stackoverflow.com/questions/115983/how-can-i-add-a-blank-directory-to-a-git-repository) like dropping a readme file there or something.

Reply:
or alternatively, `.gitkeep`